### PR TITLE
Exception if hastic webhook url is undefined #593

### LINKED ISSUE #593: Don't throw exception if HASTIC_WEBHOOK_URL is undefined
This is example:

```
Sending info webhook "Analytics is up"
(node:1141) UnhandledPromiseRejectionWarning: Error: Can't send alert, HASTIC_WEBHOOK_URL is undefined
    at /mnt/c/Projects/hastic-server/server/dist/server.js:22:40649
    at Generator.next (<anonymous>)
    at /mnt/c/Projects/hastic-server/server/dist/server.js:22:40460
    at new Promise (<anonymous>)
    at r (/mnt/c/Projects/hastic-server/server/dist/server.js:22:40237)
    at c (/mnt/c/Projects/hastic-server/server/dist/server.js:22:40579)
    at Object.<anonymous> (/mnt/c/Projects/hastic-server/server/dist/server.js:22:41787)
    at Generator.next (<anonymous>)
    at /mnt/c/Projects/hastic-server/server/dist/server.js:22:40460
    at new Promise (<anonymous>)
```

### PR DESCRIPTION
**Pull request by amper43**
fix #593

### Changes
- `console.log` instead of `throw`
- log notification message

### CODE REVIEW

File: server/src/services/alert_service.ts [hunk 1]
```diff
@@ -19,7 +19,9 @@ export class Alert {
 
   protected async send(segment) {
     const notification = await this.makeNotification(segment);
-    sendNotification(notification);
+    sendNotification(notification).catch(error => {
```
Comment (rozetko) [inline server/src/services/alert_service.ts]: Let's use `await` and `try / catch` instead

File: server/src/services/notification_service.ts [hunk 2]
```diff
@@ -42,8 +42,9 @@ export declare type Notification = {
 }
 
 export async function sendNotification(notification: Notification) {
-  if(HASTIC_WEBHOOK_URL === null) {
-    throw new Error(`Can't send notification, HASTIC_WEBHOOK_URL is undefined`);
+  if(HASTIC_WEBHOOK_URL === null || HASTIC_WEBHOOK_URL === undefined) {
```
Comment (rozetko) [inline server/src/services/notification_service.ts]: Can it be `undefined`?

File: server/src/services/notification_service.ts [hunk 3]
```diff
@@ -42,8 +42,9 @@ export declare type Notification = {
...
+    console.log(`HASTIC_WEBHOOK_URL not set, skip follow notification: ${notification.message}`);
```
Comment (rozetko) [inline server/src/services/notification_service.ts]: Maybe something like this?
```suggestion
    console.log(`HASTIC_WEBHOOK_URL is not set, skip sending notification: ${notification.message}`);
```

File: server/src/services/alert_service.ts [hunk 4]
```diff
@@ -195,7 +197,9 @@ export class AlertService {
       from: now,
       to: now
     }
-    sendNotification({ message, meta: infoAlert });
+    sendNotification({ message, meta: infoAlert }).catch(error => {
```
Comment (jonyrock) [inline server/src/services/alert_service.ts]: ad also `await` here

**Pull request merged**